docs/pipeline.md

```diff
@@ -79,9 +82,11 @@ services:
     depends_on:
       component_2:
         condition: service_completed_successfully
-    image: component_3:latest
+    build: ./component_3
 ```
 
+Note that for components that do not come from the registry (local custom components) the compiler will add a build subsection instead (see component_3 in the example above) of referring to the image specified in the `component_spec.yaml`. This allows docker-compose to build and rebuild the container used in that component allowing for quicker iteration.
```

Comment:
```suggestion
Note that for components that do not come from the registry (local custom components) the compiler will add a build subsection (see component_3 in the example above) instead of referring to the image specified in the `component_spec.yaml`. This allows docker-compose to build and rebuild the container used in that component allowing for quicker iteration.
```